Packs/VMwareVCenter/ModelingRules/VMwareVCenterModelingRules/VMwareVCenterModelingRules.xif

```diff
@@ -0,0 +1,88 @@
+| alter Hour = arrayindex(regextract(_raw_log ,"\d+\-\d+\-\d+.(\d+\:\d+\:\d+)") ,0)
+| alter TimeStamp = concat(Date," ",Hour)
+| alter LogLevel = arrayindex(regextract(_raw_log ,"\d+\-\d+\-\d+\w\d+\:\d+\:\d+\.\d+\w+\s\|\s+(\S+)\s\s\|") ,0)
+| alter LogLeve2 = arrayindex(regextract(_raw_log ,"\[\w+\]\[(\w+)"),0)
+| alter LogLeve3 = arrayindex(regextract(_raw_log ,"\[\w+\-\d+\-\w+\-\S+\s(\w+)"),0)
+| alter LogLeve = coalesce(LogLevel ,LogLeve2, LogLeve3 )
+| alter Message0 = arrayindex(regextract(_raw_log ,"\|\s\d+\s\|\s([^\t]+)") ,0)
+| alter Message1 = arrayindex(regextract(_raw_log ,"\]\s([^\t]+)") ,0)
+| alter Message2 = arrayindex(regextract(_raw_log ,"\|\s\S+\s\S+\S\s\<\S+([^\.]+)") ,0)
+| alter Message = coalesce(Message0, Message1,Message2)
+| alter Description1 = arrayindex(regextract(_raw_log ,"\|\s\w+\-\w+\-\d+\s+\|\s([A-Z][^\!]+)") ,0)
+| alter Description2 = arrayindex(regextract(_raw_log ,"\[\d+\][A-Z]+\S+\:([^\!]+)") ,0)
+| alter Description3 = arrayindex(regextract(_raw_log ,"([A-Z]{7}[^\!]+)") ,0)
+| alter Description= coalesce(Description1 ,Description2,Description3 )
+| alter msg = arrayindex(regextract(_raw_log ,"INFO[^\|]+\|\s\S+\s+\|\s\w+\s+\|\s+([^\\]+)") ,0)
+| alter msg2= arrayindex(regextract(_raw_log ,"[endpoint|vmon]\s\-\s\-\s\-\s+([^\@]+)") ,0)
+| alter msg1= concat(msg ,message,LogLeve ,Description,msg2  )
+| alter originator=arrayindex(regextract(_raw_log ,"Originator\@(\d+)") ,0)
+| alter sub = arrayindex(regextract(_raw_log ,"(\w+)\sopID") ,0) 
+| alter opID= arrayindex(regextract(_raw_log ,"opID\=([^\]]+)") ,0)
+| alter XDM.Audit.event_timestamp =parse_timestamp("%Y-%m-%d %H:%M:%E3S",TimeStamp)
+, XDM.Audit.threat.description = Message
+| alter XDM.Audit.audited_resource.id =opID
+| alter XDM.Audit.TriggeredBy.identity.uuid= originator 
+| alter XDM.Audit.identity.sub_type= sub 
+, XDM.Audit.identity.name = LogLevel;
+
+
+[MODEL: dataset ="VMware_vCenter_raw", model=Auth] 
+filter _raw_log contains "vpxd" or _raw_log contains "[vmafdd]" or _raw_log contains " vmdird "
+| alter Date = arrayindex(regextract(_raw_log ,"(\d+\-\d+\-\d+).\d+\:\d+\:\d+") ,0)
+| alter Hour = arrayindex(regextract(_raw_log ,"\d+\-\d+\-\d+.(\d+\:\d+\:\d+)") ,0)
+| alter TimeStamp = concat(Date," ",Hour)
+| alter Message_vpxd= arrayindex(regextract(_raw_log ,"vpxd\[\d+\](.*)") ,0)
+| alter user0_vpxd= arrayindex(regextract(_raw_log ,"user\s\<([^\>]+)") ,0)
+| alter user1_vpxd= arrayindex(regextract(_raw_log ,"User\:\s'([^'])") ,0)
+| alter Message_vmdrid= arrayindex(regextract(_raw_log ,"@\d+\:\s([^\t]+)") ,0)
+| alter Message_vmafdd= arrayindex(regextract(_raw_log ,"vmafdd.*\]\s[^\:]+\:\s([^\t]+)") ,0)
+| alter eventName_vmafdd = arrayindex(regextract(_raw_log ,"vmafdd.*\]\s([^\:]+)") ,0)
+| alter Message = coalesce(Message_vpxd ,Message_vmdrid ,Message_vmafdd )
+| alter user = coalesce(user0_vpxd , user1_vpxd )
+| alter XDM.Auth.threat.description = Message
```

Comment:
Please change XDM.Auth.threat.description to  XDM.Auth.original_event_description